libs/entry-components/dialog/dialogs/entry-dialog.component.html

```diff
@@ -1,8 +1,10 @@
-<div class="entry-dialog">
+<div class="entry-dialog" role="dialog">
     <div class="dialog-header">
-        <h1 class="title" [ngClass]="{'without-close-icon': hideClose === true}" mat-dialog-title>{{ title }}</h1>
-        <button mat-icon-button *ngIf="!hideClose" type="button" class="close-button" (click)="cancel()">
-            <mat-icon>close</mat-icon>
+        <h1 class="title" tabindex="0" [ngClass]="{'without-close-icon': hideClose === true}" mat-dialog-title>{{ title
```

Comment:
So title should be focusable?

Reply:
no, I will remove that, screen readers will read title if we have role, etc, and with keyboard, titles generally don't require user interaction